openxla_patches/f16_abi_clang.diff

```diff
@@ -1,19 +0,0 @@
-upstream CI will fail without this
```

Comment:
Do you know why we were able to remove this patch? Is it because we updated the compiler in the CI?

Reply:
I think we need to kick off upstream CI build targetting this branch and see whether CI will pass

Reply:
yeah turns out I do still need those patches... otherwise the training job hangs.